### DISCUSSION TITLE
How is it open source ?

### DISCUSSION THREAD

**Discussion by khanakia** [pavel-odintsov/fastnetmon#1012] [Q&A] [answered]
I see fastnetmon has bandwidth-based pricing https://fastnetmon.com/price/

So how is it open source? I can't get it

Would anybody help me to explain this ?

**Comment by pavel-odintsov** [comment, marked as answer]
Hello!

Thank you for question!

There are two editions of our product: Community (open source) and Advanced (commercial, proprietary code).

You can find whole list of differences between them here: https://fastnetmon.com/compare-community-and-advanced/
